networkx/generators/tests/test_create_using.py

```diff
@@ -0,0 +1,48 @@
+import pytest
```

Comment:
It looks like these tests are the same as the ones in `test_random_graphs.py`? If so, they should be removed here for the sake of deduplication.

Reply:
Yeah, not sure how I ended up with that. I've removed the dedicated file.